operatorconfig/driverconfig/common/k8s-1.24-values.yaml

```diff
@@ -24,7 +24,7 @@ images:
   externalhealthmonitorcontroller: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.11.0
 
   # "images.sdcmonitor" defines the container images used to monitor sdc container
-  sdcmonitor: dellemc/sdc:4.5.1
+  sdcmonitor: dellemc/sdc:4.5.2.1
```

Comment:
CSM 1.11.0 supports only k8 1.28-1.30, so I think there is no need to update the version here.
But please make sure to check that what is the under neath k8 version for OCP 4.15 and 4.16.

Thanks